src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -3620,9 +3679,9 @@ public class TraceJittedMethod
         /// </summary>
         public int ThreadID;
         /// <summary>
-        /// Indication of if it was JIT'd in the background
+        /// Indication of if it was JIT'd in the background and why
         /// </summary>
-        public bool IsBackGround;
```

Comment:
Just want to check - removing this property is OK? This is a public property on a public type. If TraceEvent has similar versioning rules to other BCL libraries removing public surface area would be bad news, but I wasn't sure what rules TraceEvent was operating under.

Reply:
While we have wiggle room to break things, I don't like to use it unnecessarily.   SInce it is so easy not to break things, that is my recommendation.  We like to leave things there but mark them Obsolete to be clear it may go away in the future.  

Thus put the IsBackground property back (as a property calling CompilationThreadKind != ForeGround), mark it as [Obsolete("Use CompilationThreadKind")], and you are done.